hw/ip/tlul/generic_dv/xbar_sim_cfg.hjson

```diff
@@ -0,0 +1,165 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the name of the DUT.
+  name: xbar
+
+  // Top level testbench name (sv module).
+  tb: tb
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:{dut}_sim:0.1
```

Comment:
Add comment indicating that `dut` is set in the autogenerated class.

Reply:
Updated.